lerobot/common/datasets/push_dataset_to_hub/pusht_zarr_format.py

```diff
@@ -222,17 +239,21 @@ def from_raw_to_lerobot_format(
     video: bool = True,
     episodes: list[int] | None = None,
 ):
+    # Manually change this to True to not include images at all (but don't merge with True). Also make sure to
+    # use video = 0 in the `push_dataset_to_hub.py` script.
+    keypoints_only = False
```

Comment:
FYI, I had the idea to add a **kwargs style interface in `push_dataset_to_hub.py` but it added too much complication for this one option. I think we can leave this manual for now and consider dataset specific kwargs later.